spec/lib/mail_handler/mail_handler_spec.rb

```diff
@@ -336,6 +336,26 @@ def expect_header_string(fixture_file, header, header_string)
 
 describe 'when getting attachment attributes' do
 
+  it 'should handle an Outlook attachment with HTML generated from RTF' do
+    mail = get_fixture_mail('outlook-encoding-rtf.email')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.